src/js/tracking-you/index.js

```diff
@@ -298,18 +298,10 @@ export default function setupAnalytics() {
       
       // Setting up trackers for big blue bar chart
       document.addEventListener('setup-sd-tab-tracking', function() {
-        let large_tab_classes = ['income','housing','healthcare'];
-        large_tab_classes.forEach(tlabel => {
-          let selector = "button.large-tab." + tlabel;
-          console.log("Setting up",selector);
-          document.querySelectorAll(selector).forEach(btn => {
-            console.log("Found",selector);
-            btn.addEventListener('click', function (e) {
-              reportGA('tab-select', tlabel);
-
-            });
-          });
-        });
+        for (let tlabel of ['income','housing','healthcare']) {
+          const btn = document.querySelector("button.large-tab." + tlabel);
+          btn.addEventListener('click', (e) => reportGA('tab-select', tlabel) );
```

Comment:
seems like this is vulnerable to an error if the element is not found when the other version wasn't because the selector is defined inside the loop and btn could be undefined when you try to access .addEventListener. Could add a check to make sure it exists first

Reply:
Sounds good.